lxd/container_lxc.go

```diff
@@ -7749,7 +7792,7 @@ func (c *containerLXC) removeUnixDevices() error {
 	// Go through all the unix devices
 	for _, f := range dents {
 		// Skip non-Unix devices
-		if !strings.HasPrefix(f.Name(), "unix.") && !strings.HasPrefix(f.Name(), "infiniband.unix.") {
+		if !strings.HasPrefix(f.Name(), "forkmknod.unix.") && !strings.HasPrefix(f.Name(), "unix.") && !strings.HasPrefix(f.Name(), "infiniband.unix.") {
```

Comment:
Why are you making this change? If the container shutsdown and there's a leftover forkmknod.unix file, don't we want it deleted?

Reply:
I don't understand. This is what this is doing.

Reply:
Oh, misread, nevermind